src/util/languageUtils.js

```diff
@@ -38,16 +38,10 @@ export const getLanguage = (
 	request,
 	supportedLangs,
 	defaultLang = LANG_DEFAULT
-) => {
-	// return the first language hit in the order of preference
-	supportedLangs.sort(l => l !== defaultLang);
```

Comment:
>given that the 'verbose' alternative is supportedLangs.sort(l => l === defaultLang ? -1 : 1);, I'd prefer 'correct' over terse.

but your 'correct' version also unnecessarily rearranges the rest of the list which is extremely confusing unless its been directly addressed. It appears to be a side effect but I only could have guessed...and hoped that the test suite complete.